articles/machine-learning/how-to-troubleshoot-online-endpoints.md

```diff
@@ -129,6 +130,12 @@ Below is a list of common deployment errors that are reported as part of the dep
 * [OperationCancelled](#error-operationcancelled)
 * [InternalServerError](#error-internalservererror)
 
+### ERROR: ImageBuildFailure
```

Comment:
Thanks Chifeng for this. My feedback for rewrite below - pls review. @Blackmist can certainly make it user facing

This is caused due to errors during environment build. You can check the build log to diagnose  the problem. Build log path is included in error message that is returned, for e.g. : 'The build log is available in the workspace blob store "your-blob-storage" under the path "/azureml/ImageLogs/your-image-id/build.log"'.

In some cases, not pinning the version dependencies in your conda file might lead to non obvious error message: the last line of the log file contains something similar to 'Installing pip dependencies: ...working...'. 
Pinning version dependencies in your conda file could fix this issue.

It is a good idea to use [local deployment](#deploy-locally) to test and debug your models locally before deploying to the cloud.